src/octopeer-github/test/main/Database/RESTApiDatabaseAdapterTest.ts

```diff
@@ -135,4 +135,17 @@ describe("A RESTApiDatabaseAdapter", function() {
         expect(consoleSpy).toHaveBeenCalledTimes(2);
         expect(consoleSpy.calls.all()[0].args[0]).toBe("Constructed DatabaseAdapter(http://localhost:8000/)");
     });
+
+    it("has to round data that is only accepted as integers", function() {
+        let joc = jasmine.objectContaining;
+
+        adapter.post(EventFactory.mouseScroll(23.1, 12.8, 123.456), spyFunc, EMPTY_CALLBACK);
+        let expected = (<any>jasmine.Ajax.requests.mostRecent()).params;
+
+        expect(JSON.parse(expected)).toEqual(joc({
```

Comment:
Do we still need the `joc` alias or can you just turn this back into `jasmine.objectContaining`?